app_dart/lib/src/model/github/create_event.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Sounds a good upstream support.

Reply:
+1 - Please upstream to package:github and add me as a reviewer

Reply:
For sure, would be great if you can help review whether create_event.dart looks decent with the current three fields. If it looks alright I will raise a pr at https://github.com/SpinlockLabs/github.dart/blob/5aa76d272341df4b8dc6c41cd6daf397ff0f0e7d/lib/src/server/hooks.dart

Reply:
I think you want to add the type and pusher (generally, anything in the event json should be added)

Reply:
sure, added type and sender